src/selectVendorByNameOrAlias.js

```diff
@@ -1,28 +1,26 @@
-const fs = require('fs');
-const Path = require("path");
+const vendors = require('./vendors');
 
-const vendorsPathPrefix = Path.join(__dirname, "..", "vendors");
-const vendorFiles = fs.readdirSync(vendorsPathPrefix);
-
-const vendorAliasMap = new Map();
-const filesAlreadyCached = new Set();
+const vendorAliasMap = createVendorPointerMap(vendors);
```

Comment:
Updated the script to statically import the vendor files instead of dynamically.